.circleci/config.yml

```diff
@@ -3303,14 +3303,14 @@ workflows:
           slack-mentions: "${SLACK_USER_ID}"
 
   nightly:
-    triggers:
-      - schedule:
-          cron: "0 5 * * *"
-          filters:
-            branches:
-              only:
-                - main
-                - /release-(\d)+(\.(\d)+)+(-ee)?/
+    # triggers:
+    #   - schedule:
+    #       cron: "0 5 * * *"
+    #       filters:
+    #         branches:
+    #           only:
+    #             - main
+    #             - /release-(\d)+(\.(\d)+)+(-ee)?/
```

Comment:
This was done to get the nightlies to run, but will be uncommented before merging.

Reply:
Love the idea. I'd never thought of it myself.

I added instructions for same to the [CI triage page](https://hpe-aiatscale.atlassian.net/wiki/spaces/ENGINEERIN/pages/1306132589/CI+triage+runbook#Forcing-nightly-tests-to-run-in-development). Let me know what you think and/or please edit as necessary.

It's probably a pain, but just for ultra safety it'd be nice to leave this uncommented when up for review. I think it could be commented out to force execution, then uncommented again when up for review, providing a link to the executed workflow (this time, [here](https://app.circleci.com/pipelines/github/determined-ai/determined/44031/workflows/db7a736b-4a90-4a85-ab49-bdaf89dec53e)).